src/js/engagement_view/src/services/graphQLRequests/utils_retrieveGraph/expandLensScopeQuery.tsx

```diff
@@ -7,23 +7,27 @@ export const expandLensScopeQuery = (lensName: string) => {
             lens_type,
             dgraph_type,
             score,
+            display,
             scope {
                 ... on Process {
                     uid,
                     node_key, 
                     dgraph_type,
                     process_name, 
                     process_id,
+                    display,
                     children {
                         uid, 
                         node_key, 
                         dgraph_type,
+                        display,
                         process_name, 
                         process_id,
                     }, 
                     risks {  
                         uid,
                         dgraph_type,
+                        display,
```

Comment:
Not sure if risks actually has a display or not, but it probably doesn't need one.

Reply:
Removed